CHANGELOG.md

```diff
@@ -259,19 +275,21 @@ If you are upgrading, please see the [upgrade guide](UPGRADING.md).
 - Update `level-packager` to `~0.18.0` ([**@rvagg**](https://github.com/rvagg))
 - Update `leveldown` to `~0.10.0` ([**@rvagg**](https://github.com/rvagg))
 
-## [0.17.0-1] - 2013-10-09
+## [0.17.0] - 2013-10-01
 
 ### Changed
 
-- Use `level-packager` instead of `levelup` ([**@rvagg**](https://github.com/rvagg))
-- Run tests in `level-packager` using `tape` ([**@rvagg**](https://github.com/rvagg))
+- Update `levelup` to `~0.17.0` ([**@rvagg**](https://github.com/rvagg))
+- Update `leveldown` to `~0.9.0` ([**@rvagg**](https://github.com/rvagg))
 
-## [0.17.0] - 2013-10-01
+**Historical Note** 0.17.0 and 0.17.0-1 are listed out of order here, due to 0.17.0-1 not adhering to the semver rules that we follow today.
```

Comment:
FYI, the reason these sections got reordered is that `hallmark --fix` sorts releases by semver rules.